_posts/2019-01-09-disobey-puzzle-with-powershell.md

```diff
@@ -0,0 +1,283 @@
+---
+layout: post
+title: Solving Disobey 2019 puzzle with PowerShell!
+author: Rinorragi
+excerpt: I was able to claim my hacker badge for Disobey 2019 mainly using PowerShell.
+tags:
+- disobey
+- hacking
+- infosec
+- software security
+- devsecops
+- PowerShell
+---
+
+## Disobey 
+
+[Disobey](https://disobey.fi/) is a nordic security event. It is all about gathering people from different organizations around the security topic. The atmosphere in the disobey is hacker friendly. Many are interested on the techniques to attack systems. Most of the people are white hat hackers that try to find vulnerabilities that could be then patched or limited with other measures. We have been participating Disobey in [2017](https://dev.solita.fi/2017/01/19/Disobey.html) and [2018](https://dev.solita.fi/2018/01/26/Disobey.html).
```

Comment:
"in the disobey" -> "in Disobey".
"interested on " -> "interested in".
"white hat hackers that try" -> "white hat hackers who try".
"We have been participating Disobey in" -> "We have been at Disobey in 2017.."